src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -241,10 +245,25 @@ void GcsPlacementGroupScheduler::CancelResourceReserve(
 
   return_client->CancelResourceReserve(
       *bundle_spec,
-      [bundle_spec, node_id](const Status &status,
-                             const rpc::CancelResourceReserveReply &reply) {
-        RAY_LOG(DEBUG) << "Finished cancelling the resource reserved for bundle: "
-                       << bundle_spec->DebugString() << " at node " << node_id;
+      [this, bundle_spec, node_id, node, max_retry, current_retry_cnt](
```

Comment:
Where do you use the max retry

Reply:
oops.. let me fix this. I will also add an unit test